lib/oxide-vpc/tests/common/mod.rs

```diff
@@ -95,6 +97,12 @@ pub const VPC_LAYERS: [&str; 5] =
 pub const GW_MAC_ADDR: MacAddr =
     MacAddr::from_const([0xA8, 0x40, 0x25, 0xFF, 0x77, 0x77]);
 
+pub const BS_MAC_ADDR: MacAddr =
+    MacAddr::from_const([0xA8, 0x40, 0x25, 0x77, 0x77, 0x77]);
```

Comment:
This falls into the physical devices space of our [MAC address space](https://github.com/oxidecomputer/oana#mac-addresses). We also have some further constraints on the software-defined/virtual space noted in [omicron](https://github.com/oxidecomputer/omicron/blob/eee942f601bbebf00352eb5c5f060e1e80a437e6/common/src/api/external/mod.rs#L1870-L1887).

This is just for tests but `FF:00:00-FF:7F:FF` would be where we can draw from for fixed addresses (i.e. `GW_MAC_ADDR`).

I also realize now that the [boundary services mac addr](https://github.com/oxidecomputer/omicron/blob/eee942f601bbebf00352eb5c5f060e1e80a437e6/illumos-utils/src/opte/mod.rs#L46) we're using also conflicts :/  It falls into the guest addresses range (`F0:00:00-FE:FF:FF`) so an instance could potentially get assigned the same address. Opened issue for that https://github.com/oxidecomputer/omicron/issues/3984

Reply:
Thinking on it a lil more, can we move `GW_MAC_ADDR`, `BS_MAC_ADDR`, `BS_IP_ADDR`, `BOUNDARY_SERVICES_VNI` to be exported from `oxide-vpc` to `omicron` can also use them leaving us with less places to have to keep in sync.

Reply:
- I've moved `GW_MAC_ADDR` to `oxide_vpc::api`
- Redefined `BS_MAC_ADDR` in terms of `oxide_vpc::engine::overlay::INTERNET_GATEWAY_MAC`
- `BS_IP_ADDR` is staying put, as that's no longer a static entity, but we need to choose something for the purpose of this test.